src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+import $style from './slider-improvements.module.css';
+import SliderButton from '@src/features/basic/slider-improvements/SliderButton.vue';
+import { clickElement } from '@src/util';
+
+function onTileReady(tile: PrunTile) {
+  sliderMinMaxButtons(tile);
+  sliderTableBulkMinMax(tile);
+}
+
+function sliderTableBulkMinMax(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    const headerRow = _$(table, 'thead')?.firstElementChild;
```

Comment:
Avoid using `_$`. Instead, make the function to `async` and use `await $` - it will give you a non-undefined element.